cognite_toolkit/cdf_tk/utils.py

```diff
@@ -315,6 +315,28 @@ def verify_dataset(self, data_set_external_id: str) -> int:
             f"Data set {data_set_external_id} does not exist, you need to create it first. Do this by adding a config file to the data_sets folder."
         )
 
+    def verify_extraction_pipeline(self, external_id: str) -> int:
+        """Verify that the configured extraction pipeline exists and is accessible
+
+        Args:
+            external_id (str): External id of the extraction pipeline to verify
+        Yields:
+            extraction pipeline id (int)
+            Re-raises underlying SDK exception
+        """
+
+        self.verify_client(capabilities={"extractionPipelinesAcl": ["READ", "WRITE"]})
```

Comment:
You don't need write to do a retrieve
```suggestion
        self.verify_client(capabilities={"extractionPipelinesAcl": ["READ"]})
```